requirements.txt

```
@@ -3,7 +3,7 @@ torchvision>=0.10.1
 Pillow>=7.1.2
 PyYAML>=5.3.1
 gcsfs
-luxonis-ml[data,nn_archive,utils]>=0.6.1
+luxonis-ml[data,nn_archive,utils]==0.6.4
```

I think we can make it like this so it doesn't need to be updated on every release and we make sure that we don't do breaking changes on the luxonis-ml side
```suggestion
luxonis-ml[data,nn_archive,utils]>=0.6.4,<0.7
```